### PR TITLE
Refactor of Wazuh Dockerfile

### PR DESCRIPTION
**Pull request by manuasir**
Hi team,

This PR deletes an unused file and also refactors the `Dockerfile` for the `Wazuh` image.
Related issue: #116 

Cheers

### CODE REVIEW

File: wazuh/Dockerfile [hunk 1]
```diff
@@ -3,53 +3,31 @@ FROM phusion/baseimage:latest
...
-ADD config/init.bash /init.bash
+# Set repositories.
+RUN set -x && echo "deb https://packages.wazuh.com/3.x/apt/ stable main" | tee /etc/apt/sources.list.d/wazuh.list && \
+   curl -s https://packages.wazuh.com/key/GPG-KEY-WAZUH | apt-key add - && \
+   curl --silent --location https://deb.nodesource.com/setup_8.x | bash - && \
+   echo "postfix postfix/mailname string wazuh-manager" | debconf-set-selections && \
+   echo "postfix postfix/main_mailer_type string 'Internet Site'" | debconf-set-selections && \
+   groupadd -g 1000 ossec && useradd -u 1000 -g 1000 ossec
+
+RUN apt-get update && apt-get upgrade -y -o Dpkg::Options::="--force-confold" && apt-get --no-install-recommends --no-install-suggests -y install openssl postfix bsd-mailx python-boto python-pip  \
```
Comment (hatityechindove) [inline wazuh/Dockerfile]: Are we going to be able to support `boto3` with this option?

**Comment by LuisGi93** [comment]
Hi team,
this PR is been tested right now. During the test some fixes were needed and a [PR](https://github.com/wazuh/wazuh-docker/pull/126) was done to add the fixes.

The following test (at least) are spected to be done:

- [ ] Full deploy. 
- [ ] Agent registration. 
- [ ] Wazuh manager updates. 
- [ ] Register an agent, update the manager and register another agent. Need to check for database templates errors.
- [ ] Check volume mounts works.

**Comment by LuisGi93** [comment]
Update: the test were done.

File: wazuh/Dockerfile (hunk 1 above)
Comment (manuasir) [reply wazuh/Dockerfile]: Hi @hatityechindove , that dependency along with `pip` are needed for the AWS integration, so events can be pulled from Wazuh. Check further information [here](https://documentation.wazuh.com/current/amazon/configuration/dependencies.html). Thanks.

**Pull request merged**